apps/nextjs-app-router/src/components/group-demo/group-demo.makeswift.ts

```diff
@@ -4,31 +4,31 @@ import { lazy } from 'react'
 import {
   Style,
   TextInput,
-  Shape,
+  Group,
```

Comment:
Here and below, renaming `Shape` to `Group` in the `app` dir examples, no semantic changes.